.circleci/config.yml

```
@@ -213,3 +216,26 @@ workflows:
               only: master
           requires:
           - deploy
+  push-to-app-catalog:
+    jobs:
+      - architect/push-to-docker:
+          image: "quay.io/giantswarm/chart-operator"
+          username_envar: "QUAY_USERNAME"
+          password_envar: "QUAY_PASSWORD"
+          requires:
+            - build
+          filters:
+            tags:
+              only: /^v.*/
+      - architect/push-to-app-catalog:
+          name: "package and push chart-operator-chart"
+          app_catalog: "giantswarm-catalog"
+          app_catalog_test: "giantswarm-test-catalog"
+          chart: "chart-operator-chart"
```

You'll need to prepare a new chart. See https://gigantic.slack.com/archives/C04TGHDEF/p1559728459002200

```suggestion
          chart: "chart-operator"
```

---

@tomahawk28 I don't see the 2nd chart in the PR. Don't we need this?

---

ah, this is the reason why I failed as below. 
```
mkdir build && helm package ./helm/chart-operator --destination ./build 00:00
Exit code: 1
#!/bin/bash -eo pipefail
mkdir build && helm package ./helm/chart-operator --destination ./build
Error: chart metadata (Chart.yaml) missing
Exited with code 1
```

---

Damm, it blocked by gitignore rule that we put on 😆 
```
  4 # build result
  5 chart-operator
  6 

```